department-of-veterans-affairs/caseflow
# Reset virtual hearing states on cancel

Resolves #15131

### Description
- reset virtual hearing state when `cancel` is clicked on conversion page to prevent virtual hearing fields from being disabled

### Testing Plan
1. Enable feature toggle on rails console `FeatureToggle.enable!(:full_page_video_to_virtual, users: [User.find_by(css_id: "BVASYELLOW")])` and `FeatureToggle.enable!(:schedule_virtual_hearings_for_central, users: [User.find_by(css_id: "BVASYELLOW")])`
2. Login as `BVASYELLOW`
3. Go to http://localhost:3000/hearings/schedule and pick a video or central hearing day
4. Click on `Virtual` on hearing type dropdown on details or convert a virtual hearing back to video or central
5. See that the conversion form opens
6. click `cancel` on lower left corner and see that the virtual hearing fields on details page are not disabled
